### PR TITLE
can't get rid of key prop error

### PR DESCRIPTION
**Pull request by tomdud9907**

### CODE REVIEW

File: src/components/TopicPage.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            comment_count,
+            author,
+            created_at,
+            votes,
+          }) => {
+            return (
+              <Col className="d-flex">
+                <Card className="productlist">
+                  <Card.Body>
+                    <Card.Title>{title}</Card.Title>
```
Comment (CatHoang) [inline src/components/TopicPage.jsx]: Do you think we would want a date too? so people can see when it was created?

**Review by CatHoang — COMMENTED** [review]
Hi Tomasz, just a few bits of feedback: 
- Once we click on a topic, should there be a way for us to get back home to all topics? Currently we cannot do this
- All your articles cards are different sizes depending on the content, they should all be the same size no matter what.
- Your topics currently have bullet points still, is this how you want to display them?
- I would start adding more styling as it is not very accessible at the moment.

File: src/components/TopicPage.jsx (hunk 1 above)
Comment (tomdud9907) [reply src/components/TopicPage.jsx]: 1,3,4 bullet done, I'll look into card sizing after lunch

Comment (tomdud9907) [reply src/components/TopicPage.jsx]: all changes applied. I need to center a Card but I'll do it later

File: src/components/TopicPage.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+          title,
+          body,
+          topic,
+          comment_count,
+          author,
+          created_at,
+          votes,
+        }) => {
+          return (
+            <Card className="text-center" style={{ width: "38rem" }}>
```
Comment (MrMcSnail) [inline src/components/TopicPage.jsx:41]: This is where you'll need a key prop to get rid of the err message

File: src/components/Nav.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    getTopics().then(({ topics }) => {
+      setTopic(topics);
+    });
+  }, []);
+
+  return (
+    <nav className="main-nav">
+      {topic.map((topic) => {
+        return (
+          <ul className="nav_bar_topics">
```
Comment (MrMcSnail) [inline src/components/Nav.jsx:18]: This `<ul>`needs to be the _parent_ element of the the topic.map( )

**Review by MrMcSnail — APPROVED** [review]
good to merge once you've seen my comments

**Pull request merged**